### DISCUSSION TITLE
Request for more style customization

### DISCUSSION THREAD

**Discussion by Nukgnaw** [Azure/api-management-developer-portal#2610] [Feature request]
Hi, I would like to know if in the future we will be able to change the style of the following element in a managed developer portal:

- The color of the toggle switch "Group by tag"
![image](https://github.com/Azure/api-management-developer-portal/assets/29558055/bbe3f35a-7d07-47a1-a6a3-381d9e383d95)

- The font color of selected operation in the list of operations
![image](https://github.com/Azure/api-management-developer-portal/assets/29558055/988ad352-8249-43c9-8abe-b619b47ff37b)

- The "Try it" button in the operation detail widget
![image](https://github.com/Azure/api-management-developer-portal/assets/29558055/cea14e52-5af8-45a6-bf42-5124043f6c2b)

- The font color of selected API in the list of API dropdown widget 
![image](https://github.com/Azure/api-management-developer-portal/assets/29558055/e9d68cbc-ac53-4e12-94af-7cf1f4d59789)

- The color of selected tag
![image](https://github.com/Azure/api-management-developer-portal/assets/29558055/0a463125-175a-477f-87ab-3bc07f74e84f)

Thanks in advance!

**Comment by ghost** [comment]
@Nukgnaw, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@Nukgnaw, thank you for requesting this feature.

**Comment by JMach1** [comment]
1. The color of the toggle switch "Group by tag" 
    - this will not be possible
3. The font color of selected operation in the list of operations 
    - it's not planned as of right now (you can only change the font color of all the operations)
4. The "Try it" button in the operation detail widget 
    - yes, will be added soon
5. The font color of selected API in the list of API dropdown widget 
    - it's not planned as of right now (you can only change the font color of all the APIs)
6. The color of selected tag 
    - it's not planned as of right now (you can only change the font color of all the tags) 

related PR which is adding some customization options (including number 4): https://github.com/Azure/api-management-developer-portal/pull/2132

**Comment by Nukgnaw** [comment]
@JMach1 Thank you!
